server/app/models/place.js

```diff
@@ -16,7 +16,7 @@ var PlaceSchema = new Schema({
     required: true
   },
   photos: [String],
-  likes: [Schema.Types.ObjectId],
+  likes: [String],
```

Comment:
And what if we will have it in future?) (we will not, of course, but still)
It's more correct to have `ObjectId` here with `ref: 'User'`, just in case.
And also, generated data is already in `ObjectId`